.github/workflows/image-push-release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Push container image
```

Comment:
Please dont add this file
we need to change this one instead

https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-macvtap-cni/1839203924632932352

Reply:
Thanks @oshoval, I see that the above Prow job uses the script [publish.sh](https://github.com/kubevirt/macvtap-cni/blob/main/automation/publish.sh) to build and push the macvtap CNI image, and it uses the make command. Are you suggesting that we can add multi-platform support(like we added for cnao nad other cnis) by make command and adding export PLATFORMS=all in publish.sh as a follow-up to the PR? This way, we would have two options: we can either continue using the Prow job [here](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-macvtap-cni/1839203924632932352) to build and publish the image, or we can switch to using GitHub Actions for the same purpose.
cc @maiqueb

Reply:
i prefer to add the "all" and stick to platform, 
switching to git actions require changes on more repos and personally won't have time to focus on that
it can be done as follow-up when time comes, but better not rush it please
we can just update prow as discussed
thanks

Reply:
Okay sure, I will raise new PR with multi support enabled using make commands so that it can be easily work with existing prow job.

Reply:
you can even add the "all" in this PR imo

Reply:
@oshoval I created a new PR: [#127](https://github.com/kubevirt/macvtap-cni/pull/127) to enable multi-platform builds using the make command. This change ensures compatibility with the existing Prow jobs that handle the build and push of release images. I will add the GitHub Actions changes in a follow-up PR.